index.md

```diff
@@ -4,49 +4,210 @@ title: Software Engineering for Busy Experts
+ 4. [Code Review](#code-review)
+ 5. [Continuous Integration](#continuous-integration)
+
+## About Thomas
+
+I have been writing code for ten years in a range of industries including science, technology, games and healthcare. In this time I have learnt _the importance of good software practices_ in writing correct code quickly, and in collaboration with others.
+
+In my time as team lead I was able to see how good software practices can _speed up delivery_ and increased quality.
+
+I am really motivated to use my experience of software engineering experts fulfil their technologies potential. Please [get in touch](mailto:hi@tkiley.co.uk) to unlock your expertise.
+
+## Course Syllabus
+
+The course consists of 5 seminars:
 
  1. [Source Control](#source-control)
  2. [Automated Tests](#automated-tests)
  3. [“Good” Code](#good-code)
  4. [Code Review](#code-review)
- 5. [Continuous Integration (CI)](#continuous-integration-ci)
+ 5. [Continuous Integration](#continuous-integration)
+
+Each of the seminars features _practical examples_ to see how these techniques can be applied. These seminars take the form of a 45 minute presentation and time for questions. This interactivity ensures the techniques can be applied in your domain.
+
+The seminars can be delivered in person at your office, via a video call or a hybrid approach.
+
+In addition to the off-the-shelf course, I am happy to tailor the course to _match your needs, experience and technologies_.
+
+**[Get in touch](mailto:hi@tkiley.co.uk) today to craft a bespoke course for your business.**
+
+I also like to foster an on-going relationship after you have completed the course to _help you put these tools and techniques into practice_.
+
+[Get in touch](mailto:hi@tkiley.co.uk) today to learn the skills needed for _effective software development_.
 
+### Source Control
 
-### Souce Control
+Source control is a tool that stores your source code, and tracks changes made to it across time. It facilitates collaboration between multiple developers working on the same code, as well as ensuring reproducible results.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet magna justo, eleifend sagittis magna pellentesque a. Vivamus nec porttitor tellus. Sed vitae volutpat diam. Nunc blandit mauris quis arcu pretium, sit amet hendrerit sapien molestie. Integer lobortis urna nec dignissim varius. Pellentesque at nunc vitae velit varius sodales at tincidunt elit. Vivamus mi sapien, tempor id lectus porta, laoreet feugiat eros. Cras vel dapibus risus, ut commodo ex. Nulla in auctor elit. Donec non placerat tellus, vitae pulvinar est. Ut eget tincidunt nisl. Curabitur congue libero eget tortor congue, laoreet gravida sem iaculis.
+**Source control is indispensable for working with other developers on code.**
+
+If you already use source control then this seminar can be tailored to whatever tools you are already using. I will cover best practices for using them successfully.
```

Comment:
Add what you will do if they're not already using it?

Reply:
You also kinda already say this later in the syllabus section, so maybe remove it - but I still think it's useful to say what you will teach in this module.

Reply:
Replace this paragraph with something that explains the seminar